src/IrcFeatures/DccChat.cs

```diff
@@ -130,8 +130,8 @@ internal override void InitWork(Object stateInfo)
 
             DccChatStartEvent(new DccEventArgs(this));
 
-            _sr = new StreamReader(Connection.GetStream(), Irc.Encoding);
-            _sw = new StreamWriter(Connection.GetStream(), Irc.Encoding);
+            _sr = new StreamReader(Connection.GetStream(), System.Text.Encoding.Default);
+            _sw = new StreamWriter(Connection.GetStream(), System.Text.Encoding.Default);
```

Comment:
This is definitely wrong since relying on the Default encoding of the OS is wrong in most cases. The library supports setting an explicit encoding as a feature. So the transport needs to expose the used encoding for the IRC connection.